Improve Https outcalls cost documentation (#5565)
dfinity/portal: docs/building-apps/essentials/gas-cost.mdx

### HTTPS outcalls

> The cycles cost for an HTTPS outcall request has a fixed and variable component.

Reviewer: ```suggestion
The cycles cost of an HTTPS outcall request has both a fixed and variable component.
```

> The fixed part accounts for the overheads associated with an HTTPS outcall, whereas the variable part charges for the resources consumed during the requests.

Reviewer: ```suggestion
The fixed component accounts for the known overhead associated with an HTTPS outcall, whereas the variable component accounts for the resources consumed during the request.
```

> The cost is scales with regard to subnet size.

Reviewer: ```suggestion
The cost scales with regard to subnet size.
```

> Just like with other functions, the cost is scaled to account for larger subnets.

Reviewer: ```suggestion
The cost is scales with regard to subnet size.
```

Currently, these fees are:

```
total_fee = base_fee + size_fee
base_fee = (3_000_000 + 60_000 * n) * n
size_fee = (400 * request_bytes + 800 * max_response_bytes) * n
```

> The `request_bytes` include not only the http body, but the entire request:

Reviewer: ```suggestion
The `request_bytes` includes the entire request, not just the HTTP body:
```
